packages/frontend/src/components/ChartConfigPanel/Legend/ReferenceLines.tsx

```diff
@@ -7,9 +7,12 @@ import {
     isNumericItem,
     TableCalculation,
 } from '@lightdash/common';
+import { debounce } from 'lodash';
```

Comment:
can you import from lodash-es or use `import debouce from 'lodash/debounce'` because I think this way it includes the whole lib to the bundle